test/xref2/v408_and_above/github_issue_587.t/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```
Otherwise, this script can be inlined into `run.t`.

Reply:
I find it quite helpful to have the script external as it's easier to run and find the output - particularly for those that output html that I'd like to look at (not this one, as it happens). Is there a better way to do this?

Reply:
My preferred way is to add commands to the `run.t` file, I usually add `$ find .` then cat some files. I do `dune runtest --auto-promote test/xref2` to increase speed (I have a binding in my editor for that).
That way I'm sure everything is run from scratch and I avoid inconsistent states.

Reply:
OK, that's fine for just seeing the content of the files but I often want to see the HTML output in the browser. I suppose that's really only when developing the test, so I can always transcribe it into the `run.t` afterwards, as I agree it's better to have it inline.